configs/config.hs

```diff
@@ -52,6 +53,12 @@ writeMailtoFile m = do
   B.writeFile fname m
   pure ""
 
+fromMail :: [Mailbox]
+fromMail =
+    [ Mailbox
+          (Just "Joe Bloggs")
+          (AddrSpec "joe" (DomainDotAtom $ "foo" :| ["test"]))
+    ]
```

Comment:
It could be useful to write an `IsString` instance for Mailbox, that just blows up if it can't be converted.
This means a runtime error when processing the configuration (so we should `deepseq` / `force` the whole configuration value to catch it at program startup) but it would be more user-friendly IMO.  LMK what you think.
(Work for a subsequent ticket IMO; and easyfix, so something a new contributor could tackle).

Reply:
Yeah I like the idea. You did actually file a ticket for configuration validation (#48), but I suppose this would be a smaller item than that. Would you mind perhaps to file a ticket?

Reply:
- https://github.com/purebred-mua/purebred-email/issues/18
- https://github.com/purebred-mua/purebred/issues/212